roc_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import (
+	"math/rand"
+	"testing"
+)
+
+func TestROC(t *testing.T) {
```

Comment:
please explain how the test cases' values were obtained (_e.g._ using `numpy`+`scipy` with such and such data and such and such functions)